03_ScienceExamples/MagellanicClouds/ExtremelyMetalPoorStarsInTheLMC.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    %matplotlib inline
</pre><p>Is this needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>